File: C#/begginer/1011.cs
using System; 

class URI {

    static void Main(string[] args) { 

        int r = int.Parse(Console.ReadLine());

        double vol = (4.0/3.0) * 3.14159 * Math.Pow(r, 3);

        Console.WriteLine($"VOLUME = {vol:F3}");

    }

}

File: Java/begginer/1011.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        
        double input = scanner.nextDouble();
        
        System.out.printf("VOLUME = %.3f\n", (4.0 / 3.0) * 3.14159 * Math.pow(input, 3));
    }
 
}
